### PR TITLE
formatting changes and added 3 new apis

### PR DESCRIPTION
**Pull request by GordieH**
fixed the go formatting in router and added local-reps, local-reps/edit, and top-reps api

### CODE REVIEW

File: go-server/go.mod [hunk 1]
```diff
@@ -8,5 +8,6 @@ require (
 	github.com/gocarina/gocsv v0.0.0-20201208093247-67c824bc04d4
 	github.com/gorilla/mux v1.8.0
 	github.com/joho/godotenv v1.3.0
+	github.com/segmentio/kafka-go v0.4.8
```
Comment (GordieH) [inline go-server/go.mod:11]: I installed kafka but commented it out when it is called in the code, let me know if it gives you trouble

File: go-server/models/models.go [hunk 2]
```diff
@@ -34,3 +34,10 @@ type UserRepMap struct {
 	UserGUID string `csv:"user_guid"`
 	RepGUID  string `csv:"rep_guid"`
 }
+
+//UserRepUpdate is the json response sent to kafka
+type UserRepUpdate struct {
+	UserGUID string
+	RepGUID  string
+	Action   string
+}
```
Comment (GordieH) [inline go-server/models/models.go:43]: this is used for kafka messages in the future

File: go-server/router/router.go [hunk 3]
```diff
@@ -35,15 +35,19 @@ func Router() *gin.Engine {
...
+		c.JSON(http.StatusOK, gin.H{
+			"message": "pong",
 		})
-	}
+	})
+
+	// new apis
+	api.GET("/local-reps", middleware.LocalRepsHandler)
+	api.POST("/local-reps/edit", middleware.EditLocalRep)
+	api.GET("/top-reps", middleware.GetTopReps)
```
Comment (GordieH) [inline go-server/router/router.go:48]: fixed formatting issue and new apis

File: go-server/go.mod (hunk 1 above)
Comment (VerteDinde) [reply go-server/go.mod:11]: This is working fine for me 👍

**Review by VerteDinde — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**